packages/build/bin/generate-apidocs.js

```diff
@@ -42,7 +42,13 @@ function run(argv, options) {
     args.push('--tsconfig', config);
   }
   if (!utils.isOptionSet(apidocsOpts, '--out', '-o')) {
-    args.push('-o', 'api-docs');
+    let out = 'api-docs';
+    if (process.env.LERNA_ROOT_PATH) {
```

Comment:
Could you please remind me: which code/script is setting LERNA_ROOT_PATH?

Reply:
See #1189